examples/python/spear_qcp/target_algorithm/scripts/genericWrapper.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python2.7
 # encoding: utf-8
 
-'''
+"""
```

Comment:
Hey, I am not sure if this file should be touched at all as this is a file that to the best of my knowledge is maintained in a different library. CC @mlindauer

Reply:
Changed it back.